src/pages/Player.vue

```diff
@@ -232,6 +234,33 @@ export default {
     },
   },
   async created() {
+    // creating a promise for the third party auth functionality
+
```

Comment:
best to remove this - a bit confusing - makes the top line seem like a description for the created function.